src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -1536,6 +1536,60 @@
         ]
       }
     },
+    "toolkitMaterialStyleNamespace": {
+      "type": "generated",
+      "generator": "switch",
+      "replaces": "$toolkitMaterialStyleNamespace$",
+      "parameters": {
+        "evaluator": "C++",
+        "cases": [
+          {
+            "condition": "(useToolkit)",
+            "value": "\n      xmlns:utum=\"using:Uno.Toolkit.UI.Material\""
+          },
+          {
+            "condition": "true",
+            "value": ""
+          }
+        ]
+      }
+    },
+    "materialStyleNamespace": {
```

Comment:
materialNamespace already exists, so it needs to be removed in place of this one